sdk/tests/wallet/burn_outputs.rs

```diff
@@ -43,7 +44,7 @@ async fn mint_and_burn_nft() -> Result<()> {
     println!("account balance -> {}", serde_json::to_string(&balance).unwrap());
     assert!(search.is_some());
 
-    let transaction = account.burn_nft(nft_id, None).await.unwrap();
+    let transaction = account.burn(Burn::new().add_nft(nft_id), None).await.unwrap();
```

Comment:
In this whole file, can remove some `Burn::new()`

Reply:
fixed